From: Tomasz Buchert <tomasz@debian.org>
Date: Wed, 28 Jan 2015 23:09:09 +0100
Subject: silence desktop-entry-lacks-keywords-entry

*.desktop files should have "Keywords" defined.
This patch provides a basic Keywords entry in English.
---
 data/synapse.desktop.in | 1 +
 1 file changed, 1 insertion(+)

diff --git a/data/synapse.desktop.in b/data/synapse.desktop.in
index 564877f..4520b11 100644
--- a/data/synapse.desktop.in
+++ b/data/synapse.desktop.in
@@ -2,6 +2,7 @@
 Type=Application
 _Name=Synapse
 _Comment=Search everything you do.
+Keywords=search;find;launch
 Exec=synapse
 Icon=synapse
 Categories=GNOME;Utility;
